Makefile

```diff
@@ -13,12 +13,17 @@ latest:
 
 nightly:
 	docker build --pull -f nightly/Dockerfile.ubuntu18.04-nightly . -t mtgupf/essentia-nightly:latest
+	docker build --pull -f nightly/Dockerfile.ubuntu18.04-tensorflow-nightly . -t mtgupf/essentia-nightly:ubuntu18.04-tensorflow
+	docker build --pull -f nightly/Dockerfile.ubuntu19.04-tensorflow-nightly . -t mtgupf/essentia-nightly:ubuntu19.04-tensorflow
```

Comment:
I'd make this a separate tensorflow target. I registered essentia-tensorflow on docker hub too, so we can tag it as that.

Reply:
You mean having `essentia-tensorflow` in a seperate docker hub repo? There is already one here https://hub.docker.com/r/mtgupf/essentia-tensorflow which I made earlier for the demos. Should we use this repo to push the tensorflow support images or in the nightly one?

Reply:
Yes, exactly. Let's use `essentia-tensorflow`